documentation/api/query/v4/environments.markdown

```diff
@@ -35,6 +41,24 @@ See [the Operators page](./operators.html)
 
 * `"name"` (string): the name of an environment
 
+### Subquery Relationships
+
+Here is a list of relationships that can be queried using implicit subqueries. For
+more information consult the documentation for [subqueries].
+
+Direct relationships:
+
+* [`factsets`][factsets]: Factsets received from this environment.
+* [`reports`][reports]: Reports received from this environment.
+* [`catalogs`][catalogs]: Catalogs received for this this environment.
+
+Transitive relationships:
```

Comment:
Lost comments from rebase from @wkalt:

```
I'm not sure the transitive/direct distinction is going to be useful to users.
```

and from @senior:

```
I was a little confused by it. Are you calling it transitive because nodes connect to facts by way of factsets?
```